pkg/lib/configreader/string.go

```diff
@@ -43,6 +43,7 @@ type StringValidation struct {
 	Prefix                               string
 	Suffix                               string
 	InvalidPrefixes                      []string
+	InvalidSuffixes                      []string
```

Comment:
nit: also add `InvalidSuffixes` to `pkg/lib/configreader/string_ptr.go` (you can search `InvalidPrefixes` and pattern-match)